inventree_kicad/serializers.py

```
@@ -304,10 +319,14 @@
             if parameter.template.name.lower() in excluded_field_names:
                 continue
 
-            is_visible = 'True' if parameter.template.name.lower().strip() in ki_visible_fields else 'False'
+            is_visible = 'True' if parameter.template.name.lower().strip() in kicad_global_field_visibility else 'False'
+
+            # Check if there is a local override
+            if kicad_local_field_visibility != None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)